examples/mic_aggregator/README.md

```diff
@@ -0,0 +1,187 @@
+# PDM Microphone Aggregator Example
+
+This example provides a bridge between 16 PDM mics to either TDM16 slave or USB Audio demo running on the explorer board. It uses a modified mic_array with multiple decimator threads to support 16 DDR mics on a single 8b input port. The example is written in 'bare-metal' and runs directly on the XCORE device without an RTOS.
+
+The decimators are configured to 48kHz PCM output. The 16 channels are loaded into a 16 slot TDM slave peripheral running at 24.576MHz bit clock or a USB Audio Class 2 asynchronous interface and are optionally amplified. The TDM build provides a simple I2C slave interface to allow gains to be controlled at run-time.
```

Comment:
In user documentation, we have established a convention that a single space should appear between a number and its units.  Hence, `48kHz` becomes `48 kHz`, `24.576MHz` becomes `24.576 MHz`, etc.  This comment applies to this line and several other ones in this file.

Reply:
I was not aware of that! Fixed.